recipes/rte-rrtmgp/LICENSE.txt

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2015-2018, Atmospheric and Environmental Research and Regents of the University of Colorado.
```

Comment:
Both the dates and the authors can/should be updated (probably the license in the original too) but this can stand for the moment.